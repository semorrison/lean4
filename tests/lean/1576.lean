example : True := by rw (config := non / sense) []
