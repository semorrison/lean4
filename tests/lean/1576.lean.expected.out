1576.lean:1:35-1:38: error: unknown identifier 'non'
1576.lean:1:41-1:46: error: unknown identifier 'sense'
